smriprep/workflows/surfaces.py

```diff
@@ -268,36 +234,131 @@ def init_surface_recon_wf(*, omp_nthreads, hires, name="surface_recon_wf"):
         (inputnode, skull_strip_extern, [('skullstripped_t1', 'in_brain')]),
         (recon_config, autorecon_resume_wf, [('use_t2w', 'inputnode.use_T2'),
                                              ('use_flair', 'inputnode.use_FLAIR')]),
-        # Construct transform from FreeSurfer conformed image to sMRIPrep
-        # reoriented image
-        (inputnode, fsnative2t1w_xfm, [('t1w', 'target_file')]),
-        (autorecon1, fsnative2t1w_xfm, [('T1', 'source_file')]),
-        (fsnative2t1w_xfm, t1w2fsnative_xfm, [('out_reg_file', 'in_lta')]),
-        # Refine ANTs mask, deriving new mask from FS' aseg
-        (inputnode, refine, [('corrected_t1', 'in_anat'),
-                             ('ants_segs', 'in_ants')]),
-        (inputnode, aseg_to_native_wf, [('corrected_t1', 'inputnode.in_file')]),
-        (autorecon_resume_wf, aseg_to_native_wf, [
-            ('outputnode.subjects_dir', 'inputnode.subjects_dir'),
-            ('outputnode.subject_id', 'inputnode.subject_id')]),
-        (fsnative2t1w_xfm, aseg_to_native_wf, [('out_reg_file', 'inputnode.fsnative2t1w_xfm')]),
-        (inputnode, aparc_to_native_wf, [('corrected_t1', 'inputnode.in_file')]),
-        (autorecon_resume_wf, aparc_to_native_wf, [
-            ('outputnode.subjects_dir', 'inputnode.subjects_dir'),
-            ('outputnode.subject_id', 'inputnode.subject_id')]),
-        (fsnative2t1w_xfm, aparc_to_native_wf, [('out_reg_file', 'inputnode.fsnative2t1w_xfm')]),
-        (aseg_to_native_wf, refine, [('outputnode.out_file', 'in_aseg')]),
-
         # Output
         (autorecon_resume_wf, outputnode, [('outputnode.subjects_dir', 'subjects_dir'),
                                            ('outputnode.subject_id', 'subject_id')]),
-        (gifti_surface_wf, outputnode, [('outputnode.surfaces', 'surfaces'),
-                                        ('outputnode.morphometrics', 'morphometrics')]),
-        (t1w2fsnative_xfm, outputnode, [('out_lta', 't1w2fsnative_xfm')]),
-        (fsnative2t1w_xfm, outputnode, [('out_reg_file', 'fsnative2t1w_xfm')]),
+    ])
+    # fmt:on
+
+    if "fsnative" not in precomputed.get("transforms", {}):
+        fsnative2t1w_xfm = pe.Node(
+            RobustRegister(auto_sens=True, est_int_scale=True), name="fsnative2t1w_xfm"
+        )
+
+        # fmt:off
+        workflow.connect([
+            (inputnode, fsnative2t1w_xfm, [('t1w', 'target_file')]),
+            (autorecon1, fsnative2t1w_xfm, [('T1', 'source_file')]),
+            (fsnative2t1w_xfm, outputnode, [('out_reg_file', 'fsnative2t1w_xfm')]),
+        ])
+        # fmt:on
```

Comment:
This was all that was needed to isolate this step, which drew attention to the fact that we're not mapping skull-strips correctly into fsnative. This should be resolved in another PR.